
# Distance factor which multiplies the achieved distance when scoring
DIST_FACTOR: 10

# Time factor which multiplies the remaining time when scoring
TIME_FACTOR: 100

# Target position - x coordinate
TARGET_X: -17.2652

# Target position - y coordinate
TARGET_Y: -24.7699

# Initial position tolerance
# User has to move INIT_TOL meters for the game to start
INIT_TOL: 0.5

# Target position tolerance
# User has to be within TARGET_TOL meters in order to finish the game
TARGET_TOL: 0.5

# Maximum angular velocity user can achieve before game stops
MAX_ANGULAR_VEL: 13.5

# Maximum linear velocity user can achieve before game stops
MAX_LINEAR_VEL: 8

# Total game time
TOTAL_TIME: 300
